### PR TITLE
[2019.3 Backport] Add support to run post-processing on each camera in the stack (#5719)

### PR DESCRIPTION
**Pull request by phi-lira**
Backport #5719

### CODE REVIEW

File: com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs [hunk 1]
```diff
@@ -310,15 +310,32 @@ void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
...
+                m_NameContent.text = "MISSING CAMERA";
+                string warningInfo = "Camera is missing";
+                EditorGUI.LabelField(rect, m_NameContent, TempContent("", warningInfo, m_ErrorIcon), errorStyle);
+
+                // We shouldn't do this!
                 // Automagicaly deletes the entry if a user has removed a camera from the scene
-                m_AdditionalCameraDataCameras.DeleteArrayElementAtIndex(index);
-                m_AdditionalCameraDataSO.ApplyModifiedProperties();
+                //m_AdditionalCameraDataCameras.DeleteArrayElementAtIndex(index);
+                //m_AdditionalCameraDataSO.ApplyModifiedProperties();
```
Comment (ellioman) [inline com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs]: Same comment here, maybe better to remove these four lines?

File: com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs [hunk 2]
```diff
@@ -310,15 +310,32 @@ void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
...
```
Comment (ellioman) [inline com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs]: Same comment here, maybe best to remove these 4 lines?

Comment (phi-lira) [reply com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs]: Fix submitted.

**Comment by phi-lira** [comment]
Not much testing with PPv2 yet, I've pulled the package to check it compiles. PPv2 doesn't support camera stacking so testing should be easier.

**Comment by phi-lira** [comment]
Looking at the code today I think some of the issues with fixed with integrated post-processing and render pass are likely to be in PPv2, we should take a look at it while doing package validation next week.

**Pull request merged**